### PR TITLE
Updates to fix broken links

### PR DESCRIPTION
**Pull request by mcarrano**
Fixing broken links found on v5 staging site - staging.patternfly.org

### CODE REVIEW

**Comment by edonehoo** [comment]
For the upgrade guide, we should update line 17 to fix those links: 

_To learn about the most significant changes in this release, read our [release highlights](/get-started/release-highlights). A detailed list of all changes can be found in our [major release notes](get-started/upgrade/release-notes)._

File: packages/documentation-site/patternfly-docs/content/design-guidelines/components/select/select.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ A select component consists of a toggle control to open and close a menu of acti
...
 
-2. **Menu:** The menu contains a list of options or values. Menus should optimally present 7 or less items for easy scanning by the user. For longer menus, items may be grouped or scrolling may be exposed by limiting the height of the menu. See the [menu component guidelines](/components/menu/design-guidelines) for more information about supported options, including use of icons, grouping, and descriptions. The Select component includes many common built-in menu variants. It’s also possible to attach a custom menu by using the [basic panel variant](/components/select/react#panel).
+2. **Menu:** The menu contains a list of options or values. Menus should optimally present 7 or less items for easy scanning by the user. For longer menus, items may be grouped or scrolling may be exposed by limiting the height of the menu. See the [menu component guidelines](/components/menus/menu/design-guidelines) for more information about supported options, including use of icons, grouping, and descriptions. The Select component includes many common built-in menu variants. It’s also possible to attach a custom menu by using the [basic panel variant](/components/menus/select/react#panel).
```
Comment (evwilkin) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/components/select/select.md:21]: I don't believe the `[basic panel variant](/components/menus/select/react#panel)` example exists anymore - it looks like it was part of the deprecated Select docs previously, but is no longer on the site.

File: packages/documentation-site/patternfly-docs/components-data.json
Comment (evwilkin) [inline packages/documentation-site/patternfly-docs/components-data.json:1]: We'll want to delete this file completely and make these updates to this file in the new location `packages/documentation-site/patternfly-docs/pages/landing-pages/components/components-data.json`

File: packages/documentation-site/patternfly-docs/content/design-guidelines/components/select/select.md (hunk 1 above)
Comment (mcarrano) [reply packages/documentation-site/patternfly-docs/content/design-guidelines/components/select/select.md:21]: Yes, I think we can just remove that sentence. OK with you @mmenestr ?

Comment (mmenestr) [reply packages/documentation-site/patternfly-docs/content/design-guidelines/components/select/select.md:21]: Yep that's fine!

**Review by tlabaj — CHANGES_REQUESTED** [review]
There are some broken links in the deprecated component table of release highlights.  Could you address the here too. They are the `options menu` and `menu`.   PageHeader components should also be links.

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 2]
```diff
@@ -125,16 +125,16 @@ As PatternFly has matured, a number of its popular components have been repeated
...
+| Deprecated component                                                            | Recommended replacement                                                    |
+|---------------------------------------------------------------------------------|----------------------------------------------------------------------------|
+| [Application launcher](/components/menus/application-launcher/react-deprecated) | [Dropdown](/components/menus/dropdown), [select](/components/menus/select) |
+| [Context selector](/components/menus/context-selector/react-deprecated)         | [Menu](/components/menus/menu)                                             |
+| [Dropdown (deprecated)](/components/menus/dropdown/react-deprecated)            | [Dropdown](/components/menus/dropdown)                                     |
+| [Options menu](/components/menus/options-menu/react-deprecated)                 | [Dropdown](/components/menus/dropdown), [select](/components/menus/select) 
+| [PageHeader](/components/page/react-deprecated)                                 | [Masthead](/components/masthead)                                           |
```
Comment (tlabaj) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:134]: nit.  This should be `page header`

**Review by tlabaj — APPROVED** [review]
LGTM!

**Review by evwilkin — APPROVED** [review]
LGTM 👍

**Review by evwilkin — APPROVED** [review]

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**